src/java.security.jgss/share/classes/org/ietf/jgss/GSSException.java

```diff
@@ -326,10 +330,7 @@ public int  getMinor(){
      */
     public String getMajorString() {
 
-        if (majorString != null)
-            return majorString;
-        else
-            return messages[major - 1];
+        return Objects.requireNonNullElseGet(majorString, () -> messages[major - 1]);
```

Comment:
Isn't this too fancy?

Reply:
It's been around since JDK 9. I like the functionality, but I hate the name. It returns the first argument if non-null, otherwise it runs the lambda expression. I'll keep the change if you don't mind - one less thing for IntelliJ to nag about.

Thanks for taking time to review this!
I've accepted two of your suggestions and will be pushing an update shortly.